### DISCUSSION TITLE
What options do I have for making more room to see more at once besides just zooming out a ton? (picture included in description)

### DISCUSSION THREAD

**Discussion by adamfreeplaneuser01** [freeplane/freeplane#1450] [Help]
Hey all. I am new to the forum and to Freeplane. I am having a lot of my nodes cut off that are below the node "parentheses". I could zoom out more but then I wouldn't be able to read anything. 

<img width="959" alt="jisdfnasdfi" src="https://github.com/freeplane/freeplane/assets/147225942/633e63de-dde3-4457-a8ce-ab124dfbcef4">

If you have any advice or suggestions, I would greatly appreciate it!

**Comment by euu2021** [comment]
You can try to make the Commas node a double side layout, and then distribute nodes around it. 

The automatic distribution will make nodes overlap (I once suggested that the overlap avoidance happens automatically: https://github.com/freeplane/freeplane/discussions/819#discussioncomment-4147923 suggestion number 2) , so you will need to do some manual work. 

It will result in something like this (node bbb is representing your node Commas):

![image](https://github.com/freeplane/freeplane/assets/77707706/8a698dd3-9f8d-4bac-a2db-6b488592fb07)


You can also use the `Navigate->Jump in` in Commas node, and then distribute the nodes over both sides.

I can also suggest this discussion: #1207

**Reply by adamfreeplaneuser01** [reply]
Thanks for your response! However, I am having a problem regarding your comment "You can try to make the Commas node a double side layout, and then distribute nodes around it."

So I selected "top to bottom at both sides, natural flow" but all the nodes are still being places on one side right next to each other. I'm sure this is just a matter of me not doing something correct, but I am not sure how to get the nodes to appear on both sides

**Reply by euu2021** [reply]
Now, you need to distribute the nodes, by dropping them on the left side of the parent node:

![javaw_unKMSr1QaV](https://github.com/freeplane/freeplane/assets/77707706/2bbfc3f5-ab20-4fcc-83c9-ea6e67ce5319)

**Reply by adamfreeplaneuser01** [reply]
Wait, so is there no way to automatically do this so that it evenly distributes amongst the two sides?

**Reply by euu2021** [reply]
If you use Jump In into node bbbb, and create the child nodes there, they will get distributed automatically, like happens with the Root.

**Comment by euu2021** [comment]
Also, you can get rid of all useless space inside and between nodes. That's how my map looks like:
![image](https://github.com/freeplane/freeplane/assets/77707706/210ad2fa-2304-47ec-a916-66831d8387db)

**Comment by EdoFro** [comment]
dear @adamfreeplaneuser01, 
> If you have any advice or suggestions, I would greatly appreciate it!

As a long time user of Freeplane, I have learned some tricks that may be helpful to you:

1. Freeplane is not a (good) diagramming tool or presentation tool. In other words: it is not the tool a Graphic designer will choose to create his diagrams
2. Freeplane is a very good tool to organize knowledge, tasks, ideas, concepts in an easy and fast way.


Therefore, I recommend when using Freeplane to focus on the content and not on how it looks. If you start moving nodes and placing them manually, you will waste a lot of time every time you have to rearrange the content.
You will lose flexibility to rearrange the content by giving it another order or hierarchy, because you will be restricted by the visual part.

3. Mind maps allow you to have both the overview of a topic and all the details in a single file. However as persons we are not able to manage all that. 
That is why the map has levels of hierarchy, so that we can add more detail as we go through the mind map's levels. It is good practice to fold the nodes you don't need at the moment, so that you can concentrate on each topic without getting distracted by the previous ones.

4. the number of children of each node is also an important factor. If you have too many, you may want to group them together and insert intermediate subcategories.

In my opinion, you have more a problem of folding/unfolding and categories/subcategories than a problem of "How to put more information on screen"

You can also use the Map Overview to have a general look at your map while you are working in a part of it 
```View->Controls->Map overview (Ctrl+O)```

Hope it helps,

edo

**Comment by Lotus49b** [comment]
Agree with Edo on using Map Overview.

Also, using Views - toggle Toolbar off or even use Fullscreen mode (f11) for more display real estate.

I also created a hotkey to get a more granular zoom level. The stock 25% increments gave text that was too small or too big.
_acceleratorFor.MindMap/Zoomup5percent_on_selected_node=F9_

**Comment by adamfreeplaneuser01** [comment]
> If you use Jump In into node bbbb, and create the child nodes there, they will get distributed automatically, like happens with the Root.

So I just tried that and it did not work, at least for whatever I did. Are you able to provide a video or something of how to do what you are describing? Thanks!

**Reply by euu2021** [reply]
Jump In into bbbb, and then create child nodes from bbbb (not as siblings of the other children of bbbb):

![javaw_Sh2S3qPdcb](https://github.com/freeplane/freeplane/assets/77707706/1d7ef314-76f0-4629-a062-9f9084b758d9)

**Reply by adamfreeplaneuser01** [reply]
I'm still not getting it to work. But I think maybe I don't have the right settings?
<img width="403" alt="jsdjkfdsjfjsdafdsa" src="https://github.com/freeplane/freeplane/assets/147225942/e2296b1d-b697-49e3-a986-e7e94b69c5e5">

**Reply by euu2021** [reply]
Did you apply the "top to bottom at both sides, natural flow" layout to the node?

The steps are:
- start with a blank map
- create child node aaa
- select node aaa
- create child node bbb
- select bbb
- apply the "top to bottom at both sides, natural flow" layout to node bbb
- use `Navigate->Jump in` to Jump In into bbb
- create child node ccc (it will be created on the right side)
- select node bbb
- create child node ddd (it will be created on the left side)
- repeat the last steps, as needed
- use `Navigate->Jump out`

It goes like that:
![nFm2mhhXC3](https://github.com/freeplane/freeplane/assets/77707706/8d83faaa-9cd1-4cbd-a56c-3e2423ed9ddd)

The resulting mindmap:

[EXAMPLE twosides.zip](https://github.com/freeplane/freeplane/files/12907060/EXAMPLE.twosides.zip)
